apps/organizations/models.py

```diff
@@ -220,9 +220,9 @@ class Meta:
 
     def __str__(self):
         value = (
-            '{organization_name} Member - {member_name} (member_email)'.format(
+            '{organization_name} Member - {member_name} {member_email}'.format(
                 organization_name=self.organization.name,
-                member_name=self.user.profile.get_full_name(),
+                member_name=self.user.get_full_name(),
```

Comment:
Please revert the changes to this file. It is not related to this PR.

HTK convention is to have an accompanying `UserProfile` that is associated with `User`.

See:
- https://github.com/hacktoolkit/django-htk/blob/4861521025c207ae95308b63881e7fadb918369b/apps/accounts/models.py#L87
- https://github.com/hacktoolkit/django-htk/blob/4861521025c207ae95308b63881e7fadb918369b/apps/accounts/models.py#L155